mmf/configs/defaults.yaml

```diff
@@ -220,3 +225,47 @@ distributed:
     # Set if you do not want spawn multiple processes even if
     # multiple GPUs are visible
     no_spawn: false
+
+# Configuration for checkpointing including resuming and loading pretrained models
+checkpoint:
+    # If checkpoint.resume is true, MMF will try to load automatically load
+    # checkpoint and state from "current.ckpt" from env.save_dir
+    resume: false
+    # `checkpoint.resume_file` can be used to load a specific checkpoint from a file
+    # Can also be a zoo key
+    resume_file: null
+    # `checkpoint.resume_best` will load the best checkpoint according to
+    # training.early_stop.criteria instead of the last saved ckpt
+    resume_bert: false
+    # `checkpoint.resume_pretrained` can be used in conjuction with `resume_file`
+    # or `resume_zoo` where you specify a checkpoint or .pth file to be loaded
+    # but it is mapped based on `checkpoint.pretrained_state_mapping`
+    resume_pretrained: false
```

Comment:
I don't understand this. Why will this be used in "conjunction" with `resume_file` or `resume_zoo`? We don't need to load a pretrained model if we are loading from a resume_file.

Reply:
Both work. This is what `load_pretrained` used to be. This is a boolean value to be used in conjunction with `resume_file` or `resume_zoo`. Basically, in your change what you did via
`zoo_requirements` can be done via this as well.